src/pages/academia.js

```diff
@@ -42,6 +43,39 @@ const StyledButton = styled(Button)`
     margin-top: ${size.large};
 `;
 
+const BodyContent = styled('div')`
+    margin: 0 auto;
+    max-width: ${CONTENT_MAX_WIDTH};
```

Comment:
For articles we use the max width `size.maxContentWidth` from `theme`, can we use that here? (It is going to be a similar layout)